questionpy_server/worker/__init__.py

```diff
@@ -178,8 +181,11 @@ async def get_attempt(
             question_state: The question the attempt belongs to.
             attempt_state: The `attempt_state` attribute of an attempt which was previously returned by
                            :meth:`start_attempt`.
-            scoring_state: Not implemented.
+            scoring_state: The scoring state returned by the package the last time this attempt was scored.
             response: The response currently entered by the student.
+            uploads: Files uploaded by the student in file upload elements (files belonging to editors are passed in
+                     `editors`.)
+            editors: The WYSIWYG editor responses are passed separately here. Validated to a model in the SDK.
```

Comment:
Ich fände es schöner, wenn der Docstring hier und in Zeile 215 einheitlich zu denen in `questionpy_common/api/question.py` ist. Die SDK-Referenz finde ich etwas verwirrend.

```suggestion
            editors: WYSIWYG editor responses entered by the student.
```

Reply:
Ja, die Beschreibung ist gänzlich unsinnig. Ich hatte ursprünglich versucht, `uploads` und `editors` hier opak als `dict[str, ...]` zu behandeln, das wurde aber im SDK zu kompliziert. Beschreibung ist jetzt aktualisiert.